### PR TITLE
Add clip-score and fid to mindone/metrics

### PR DESCRIPTION
**Pull request by chenyang23333**

### CODE REVIEW

File: examples/stable_diffusion_xl/tools/_common/clip/__init__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+"""clip init"""
```
Comment (zhanghuiyao) [inline examples/stable_diffusion_xl/tools/_common/clip/__init__.py:1]: clip 的部分是否可以先挪到 eval/clip_score/clip 中；

**Comment by zhanghuiyao** [comment]
pre-commit 代码格式问题解决下

**Review by SamitHuang — COMMENTED** [review]
CLIP-score and fid are already implemented in sd-v2/tools. Is there any difference of improvement here? 

We should put them as a common components in `mindone/`

File: mindone/metrics/tools/_common/clip/clip.py [hunk 2]
```diff
@@ -130,10 +129,7 @@ def construct(
 
         image_features = self.get_image_features(image)
         text_features = self.get_text_features(text)
-        if not is_old_ms_version("2.0.0-alpha"):
-            L2_norm_ops = partial(ops.norm, ord=2, dim=1, keepdim=True)
-        else:
-            L2_norm_ops = partial(ops.norm, p=2, axis=1, keep_dims=True)
+        L2_norm_ops = partial(ops.norm, ord=2, dim=1, keepdim=True)
```
Comment (zhanghuiyao) [inline mindone/metrics/tools/_common/clip/clip.py:132]: 为啥要删掉原有的版本控制代码

Comment (chenyang23333) [reply mindone/metrics/tools/_common/clip/clip.py:132]: 现在是不是默认mindspore版本是2.0.0以上了，官网现在最低的版本是2.1了已经，这个判断就多余了？

**Review by zhanghuiyao — APPROVED** [review]

**Pull request closed without merging**